packaging/pip_requirements.txt

```diff
@@ -1,17 +1,20 @@
 # minimal
-Pillow==8.2.0; python_version == '3.6'
-Pillow==9.0.0; python_version >= '3.7'
+Pillow==9.3.0; python_version >= '3.7' and python_version < '3.12'
+Pillow==10.3.0; python_version >= '3.12'
+torch==1.13.0; python_version == '3.7'
+torchvision==0.14.0; python_version == '3.7'
```

Comment:
So now you moved these lines back to the minimal block? :grin: I thought we agreed on having all torch lines below next to each other.